posit-dev/positron
# Variables pane UI smoke test for Python & R

### Intent

> This pull request adds a smoketest for the Positron variables pane (Python)

### Approach

> The existing Playwright infrastructure is used and expanded upon slightly to allow more direct access to the Playwright API.

> When running locally, be sure to set the following env variables according to which Python and R version you would like to use.  The variable values should match the strings in the UI for the desired versions:
> POSITRON_PY_VER_SEL
> POSITRON_R_VER_SEL

### QA Notes

> This is a test itself.  It opens Positron, selects a specific Python version, adds some variables via the console and ensures they appear in the variables pane.  It then repeats the process with R.

> Run locally with the "Launch Smoke Test" action in VSCode.
